src/runtime/virtcontainers/qemu_amd64.go

```diff
@@ -40,6 +43,8 @@ const (
 	tdxCPUFlag = "tdx"
 
 	sevKvmParameterPath = "/sys/module/kvm_amd/parameters/sev"
+
+	sevGuestOwnerProxyClient = "/opt/sev/guest-owner-proxy/gop-client.py"
```

Comment:
@jodh-intel Yes, will do

@fitzthum What do you think? Call `gop-*.py` a "quick hack" and plan to rewrite in go? This is beyond scope of CCv0

Reply:
Ideally it would all be part of the shim. `gop-client.py` isn't a particularly sophisticated piece of code. We might lose some modularity by adding the functionality into the shim directly, though. It would set the protocol for any SEV(-ES) KBS.